awx_collection/tools/templates/tower_module.j2

```diff
@@ -128,18 +128,6 @@ def main():
     # Extract our parameters
 {% for option in item['json']['actions']['POST'] %}
     {{ option }} = module.params.get('{{ option }}')
-{% if 'min_value' in item['json']['actions']['POST'][option] %}
```

Comment:
Instead of a straight up removal of these how would you feel about putting min/max/length constraints up in the description of options? I feel like it would be nice to somehow expose this through the modules instead of just letting the API gawk at a bad value? If you got these details from `ansible-doc <module name>` I'd be happy with that.

Reply:
Yeah, that sounds like a good philosophy to me. I expect it might help that you can do this on new lines in the description, with `-` list elements.

But right now we've got a lot of other stuff we want to get in, and I want to keep it moving. This is lower priority. I think it would be a lot easier to make those patches after it's merged, and maybe after we have a module generated by the tool. Then you can make the template additions, inspect the diff, and it'll be a lot more manageable that way.